library/src/test/java/bio/terra/landingzone/stairway/flight/create/resource/step/CreateAksStepTest.java

```diff
@@ -166,6 +140,26 @@ void testCostSavingInStep() throws InterruptedException {
         equalTo("true"));
   }
 
+  @Test
+  void testCostSavingVpaInStep() throws InterruptedException {
+    costSavingsVpaEnabled = "true";
```

Comment:
How about making `setupCostSavingLZ` return the mock it sets up with default behavior, and you can directly override the mock to return the value you want it to return here, inline in the test?

Reply:
(Then you can get rid of the instance variable)

Reply:
yeah, I think if I am going to do any more major test refactors, it may make sense to have all the tests follow it - for now trying to keep things consistent compared to others but I will let @sergiygetlin weight in on this since I am not aware of all the gotchas the test set up may have.